packages/devtools_shared/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   shelf: ^1.1.0
   sse: ^4.1.2
   usage: ^4.0.0
-  vm_service: ^13.0.0
```

Comment:
will this be breaking for devtools_shared dependencies? As in, is a major version bump needed for devtools_shared?

Reply:
I don't think a major version bump is required because this update won't be breaking for packages that only depend transitively on `package:vm_service` thorugh `package:devtools_shared`. Packages that depend directly on both `package:vm_service` and `package:devtools_shared` should have `package:vm_service` version constraints that will prevent them from getting this version of `package:devtools_shared` until they're ready to.